### PR TITLE
feat: add Android feature flag & change some app logos

### PR DESCRIPTION
**Pull request by kristiehuang**
Adding android GA launch feature flag! + Updating language from iOS to just app & adding color app logo & reduce space above "Learn more" in landing page

Got approval from design on deploy

_Linear ticket:_ https://linear.app/uniswap/issue/WEB-2988/lets-get-this-icon-changedupdated-now-that-we-are-on-both-platforms
_Slack thread:_
_Relevant docs:_

### Before
Screenshot 2023-10-30 at 6 58 25 PM
Screenshot 2023-10-30 at 6 31 25 PM

### After
Screenshot 2023-10-30 at 6 57 34 PM
Screenshot 2023-10-30 at 6 57 52 PM

### Reproducing the error

1. 

### QA (ie manual testing)

- [ ] N/A

### Automated testing

- [ ] Unit test
- [ ] Integration/E2E test

### CODE REVIEW

**Review by natew — APPROVED** [review]
missing after photos but looks good

**Comment by kristiehuang** [comment]
> missing after photos but looks good

yep just updated sry!

**Review by mattbspector — APPROVED** [review]
LGTM SHIPIT

File: src/components/FeatureFlagModal/FeatureFlagModal.tsx [hunk 1]
```diff
@@ -304,6 +305,12 @@ export default function FeatureFlagModal() {
           featureFlag={FeatureFlag.progressIndicatorV2}
           label="Refreshed swap progress indicator"
         />
+        <FeatureFlagOption
+          variant={BaseVariant}
+          value={useAndroidGALaunchFlag()}
```
Comment (natew) [inline src/components/FeatureFlagModal/FeatureFlagModal.tsx:310]: such a weird pattern we have going on here, you'd think we'd have something like `useFlag(Flags.AndroidGALaunch)` and then just a simpler component here like `<FeatureFlag from={Flags.AndroidGALaunch} />` or something along those lines... not saying to change it here but something to look at in the future

File: src/components/NavBar/MenuDropdown.css.ts [hunk 2]
```diff
@@ -26,6 +26,7 @@ export const MenuRow = style([
   {
     lineHeight: '24px',
     textDecoration: 'none',
+    alignItems: 'flex-start',
```
Comment (natew) [inline src/components/NavBar/MenuDropdown.css.ts:29]: does this change on flag change? i noticed you changed it in the last pr

**Review by natew — APPROVED** [review]

**Comment by natew** [comment]
I will say my eye doesn't love this - padding feels a touch cramped on right edge and feels like dark mode icon would look nicer with dark bg (cc @callil since he's the only designer who i know their github handle)

Screenshot 2023-10-31 at 12 45 31 PM

File: src/components/FeatureFlagModal/FeatureFlagModal.tsx (hunk 1 above)
Comment (kristiehuang) [reply src/components/FeatureFlagModal/FeatureFlagModal.tsx:310]: yeeaah feature flags needs improvement for sure, kinda on my backlog but i have noted your comments https://linear.app/uniswap/issue/WEB-2932/feature-flag-code-improvements

File: src/components/NavBar/MenuDropdown.css.ts (hunk 2 above)
Comment (kristiehuang) [reply src/components/NavBar/MenuDropdown.css.ts:29]: shouldn't change anything for existing rows! Needed flex-start alignment when there's both a title & subtext like in the wallet app MenuRow though

**Comment by bryankuuu** [comment]
@natew Good eye! Thanks for reviewing! That tightness you're seeing is only visible on hover. I wouldn't decrease the app logo any smaller because on light theme when the white container is less visible the pink unicorn will look too small relative to the other icons. 

The size is ok with me. let's roll with it

**Review by mattbspector — APPROVED** [review]

**Pull request merged**